service-worker/foreign-fetch/remote-server.js

```diff
@@ -0,0 +1,38 @@
+// This is the web server implementation for the "random number API".
+// It is currently deployed to https://foreign-fetch-demo.appspot.com
+
+const express = require('express');
```

Comment:
We're including a remote server example here but not a package.json file for folks to grab the dependency it uses. Worth adding?